### ISSUE TITLE
Problem if scipy version is 1.13.0

### ISSUE THREAD

**Issue by satabol** [pyvista/pyacvd#47]
Hello.
I'm user of your library. It is cool! But I have some trouble after today update. I find out that if scipy=1.13.0 then result is not good. If force reinstall to scipy=1.12.0 then result is good. Step to reproduce:

```cmd
pip.exe install pyacvd
```
![image](https://github.com/pyvista/pyacvd/assets/14288520/6464664a-fe42-48cc-8cb7-e0d9b55fdc76)

```cmd
pip.exe install --force-reinstall scipy==1.12.0
```

![image](https://github.com/pyvista/pyacvd/assets/14288520/3fa21448-61ea-4548-915e-b31ddb9eba1b)

code:

```Python
import pyacvd
from pyvista import PolyData
verts_i=[(1.0, 1.0, 1.0), (1.0, 1.0, -1.0), (1.0, -1.0, 1.0), (1.0, -1.0, -1.0), (-1.0, 1.0, 1.0), (-1.0, 1.0, -1.0), (-1.0, -1.0, 1.0), (-1.0, -1.0, -1.0)]
faces_i=[[0, 4, 6, 2], [3, 2, 6, 7], [7, 6, 4, 5], [5, 1, 3, 7], [1, 0, 2, 3], [5, 4, 0, 1]]
pdmesh = PolyData.from_regular_faces( verts_i, faces_i)
clust = pyacvd.Clustering(pdmesh)
clust.cluster(1000, maxiter=100)
remesh = clust.create_mesh()
remesh.regular_faces
```

Windows 11, Python 3.10.11

**Comment by akaszynski** [comment]
Can reproduce. Expect a fix by the end of the week.

**Comment by satabol** [comment]
Thank you! It is work now! Here is a first result:
![image](https://github.com/pyvista/pyacvd/assets/14288520/43e8d302-782e-440d-bc64-cf36cb4cb8e5)

But I install this version from source code. Can you say when one can install this version from pip repository?

P.S.
I'm one of developers of [Sverchok plugin for Blender](https://github.com/nortikin/sverchok). Here is a [docs of node](http://nortikin.github.io/sverchok/docs/nodes/spatial/mesh_clustering.html) of your pyacvd library.

**Comment by akaszynski** [comment]
This is awesome! Nice to see `pyacvd` and `pyvista` used within a plugin for blender.

Right now we're delayed due to an account issue. We're tracking it at [pypi/support #4016](https://github.com/pypi/support/issues/4016), but we might be able to have it resolved next Monday. I'll keep you informed, but expect a new release by the end of next week.

**Comment by nicolascedilnik** [comment]
I confirm that reverting scipy to 1.12 worked for me. Thanks @akaszynski for this workaround!

**Comment by akaszynski** [comment]
@satabol, this fix is now in [pyacvd v0.2.11](https://pypi.org/project/pyacvd/0.2.11/).

**Comment by satabol** [comment]
Thank you! I test it. All work fine!
